mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po

```diff
@@ -835,9 +835,6 @@ msgstr "Phone keeps ringing but no answer!"
 
 ## NOTIFICATION TRANSLATIONS ##
 # Combat notifications
-msgid "combat_fly"
```

Comment:
Is this translated in other locales? If so, please rename those too.

Reply:
Nope, and I checked.